Updated grammatically and punction errors (#115260)
MicrosoftDocs/azure-docs: articles/firewall/premium-features.md

:::image type="content" source="media/premium-features/firewall-category-search.png" alt-text="Firewall category search dialog":::

> [!IMPORTANT]

> > To use **Web Category Check** feature, user must have an access of Microsoft.Network/azureWebCategories/getwebcategory/action for **subscription** level, not resource group level.

Reviewer: ```suggestion
> To use the **Web Category Check** feature, users must have access to Microsoft.Network/azureWebCategories/getwebcategory/action for the **subscription** level, not the resource group level.
```
